### ISSUE TITLE
Create 1.3.2 release

### ISSUE THREAD

**Issue by asaj** [hyperlane-xyz/hyperlane-monorepo#2100]
We shipped the package but shoudl create a release for it

**Comment by nambrot** [comment]
I assume we can close this since @jmrossy is working on #2161

**Comment by jmrossy** [comment]
Yes, I created tags for 1.3.1 to 1.3.3 yesterday and I will indeed create a proper release for 1.3.4 today
